models/user.rb

```diff
@@ -11,16 +11,24 @@ def update_profile_image(icon)
     twitter.update_profile_image(icon)
   end
 
+  def tweet_changing_icon(icon_url)
+    twitter.update <<TWEET
```

Comment:
OH, I didn't know it! It looks String#strip_heredoc providing by ActiveSupport 😎 
I'll use it. Thank you.